applications/callflow/src/module/cf_notification.erl

```diff
@@ -0,0 +1,170 @@
+%%%-----------------------------------------------------------------------------
+%%% @copyright (C) 2017-2018, 2600Hz
+%%% @doc Sends a notification for missed call.
+%%%
+%%% <h4>Data options:</h4>
+%%% <dl>
+%%%   <dt>`send_at'</dt>
+%%%   <dd>Defines when send customer defined notification. Possible values `callflow_exec' and `channel_destroy'.
+%%%   For `callflow_exec' value notifications is send during calllfow execution.
+%%%   For `channel_destroy' value notification is send after channel(bridge) is destroed.</dd>
+%%%
+%%%   <dt>`template_id'</dt>
+%%%   <dd>Defines customer defined notification template name</dd>
+%%%
+%%%   <dt>`recipients'</dt>
+%%%   <dd>A list of JSON Objects contains different recipients for notifications.
+%%%   Each recipient object have to keys:
+%%%     <ul>
+%%%       <li>`type': Specified what kind of recipient is this. Possible values are `user' or `email'.</li>
+%%%       <li>`id': Based on `type' it could be a list of emails or user IDs (to get their email address) or
+%%%       just a single email or user ID (to get its email from).</li>
+%%%     </ul>
+%%%   </dd>
+%%% </dl>
+%%%
+%%% @end
+%%%-----------------------------------------------------------------------------
+-module(cf_notification).
+
+-behaviour(gen_cf_action).
+
+-export([handle/2]).
+-export([send_notification/3]).
+
+-include("callflow.hrl").
+
+%%------------------------------------------------------------------------------
+%% @doc maybe add termination handler then unconditionally
+%%      continue the flow
+%% @end
+%%------------------------------------------------------------------------------
+-spec handle(kz_json:object(), kapps_call:call()) -> 'ok'.
+handle(Data, Call) ->
+    case kz_json:get_ne_binary_value(<<"send_at">>, Data, <<"channel_destroy">>) of
+        <<"channel_destroy">> -> add_handler(Data, Call);
+        <<"callflow_exec">> -> send_notification(Call, [], Data);
```

Comment:
You send an empty list here but the spec of `send_notification` expects a `kz_json:object()`. I suspect the spec should have it be `kz_json:json_term()` instead?

Reply:
I looked into [cf_exe:handle_channel_destroyed](https://github.com/2600hz/kazoo/blob/master/applications/callflow/src/cf_exe.erl#L850-L854) here is is passed `kz_json:object()`.
To clean code i can replace `[]` by `kz_json:new()`.